### PR TITLE
Doctrine

### PR DESCRIPTION
**Pull request by mateusz-blachacz**
Work with doctrine

### CODE REVIEW

File: code/src/Eduweb/TrainingBundle/Controller/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Eduweb\TrainingBundle\Form\Type\RegisterType;
+
+/**
+ * Class BlogController
+ *
+ * @package Eduweb\TrainingBundle\Controller
+ * @Route("/blog/admin")
+ *
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Controller/AdminController.php]: remove this line

File: code/src/Eduweb/TrainingBundle/Controller/AdminController.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+{
+    /**
+     * @Route("/", name="edu_blog_admin_listing")
+     *
+     * @Template
+     */
+    public function listingAction()
+    {
+        $repo = $this->getDoctrine()->getRepository('EduwebTrainingBundle:Register');
+        $rows = $repo->findBy(array('sex' => ''));
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Controller/AdminController.php]: Why is there an empty value for 'sex' key?

File: code/src/Eduweb/TrainingBundle/Controller/AdminController.php [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        $rows = $repo->findAll();
+
+        return array('rows' => $rows,);
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Controller/AdminController.php]: In this case comma isn't needed here.

File: code/src/Eduweb/TrainingBundle/Controller/AdminController.php [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    /**
+     * @Route("/details/{id}", name="edu_blog_admin_details")
+     *
+     * @Template
+     */
+    public function detailsAction($id)
+    {
+
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Controller/AdminController.php]: remove empty line

File: code/src/Eduweb/TrainingBundle/Entity/Register.php [hunk 5]
```diff
@@ -3,45 +3,77 @@
...
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     *
      * @Assert\NotBlank
      * @Assert\Regex(pattern = "/^[a-zA-Z]+ [a-zA-Z]+$/", message = "Musisz podac imie i nazwisko")
+     * @Assert\Length(max=255)
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Entity/Register.php]: Is this change intentional? I don't think I know 225-character names

File: code/src/Eduweb/TrainingBundle/Entity/Register.php [hunk 6]
```diff
@@ -3,45 +3,77 @@
...
      */
     private $name;
 
     /**
+     * @ORM\Column(type="string", length=255)
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Entity/Register.php]: There is no need to declare default value. Default type is string, and default length is 255.

File: code/src/Eduweb/TrainingBundle/Entity/Register.php [hunk 7]
```diff
@@ -3,45 +3,77 @@
...
 
     /**
+     * @ORM\Column(type="string", length=2)
+     *
      * @Assert\NotBlank
      */
     private $country;
 
     /**
+     * @ORM\Column(type="string", length=255)
```
Comment (gracjan122) [inline code/src/Eduweb/TrainingBundle/Entity/Register.php]: same as above

**Review by gracjan122 — CHANGES_REQUESTED** [review]
Remember to reformat the code before pushing it, as some bugs repeat.

File: code/src/Eduweb/TrainingBundle/Controller/AdminController.php (hunk 2 above)
Comment (mateusz-blachacz) [reply code/src/Eduweb/TrainingBundle/Controller/AdminController.php]: I was testing some possibilities. I check, that, Can I find item with empty string in database. This is only test, I deleted this know because it is not necessary.

File: code/src/Eduweb/TrainingBundle/Entity/Register.php (hunk 5 above)
Comment (mateusz-blachacz) [reply code/src/Eduweb/TrainingBundle/Entity/Register.php]: This was only example for, how to youse annotation to mange database item.

File: code/src/Eduweb/TrainingBundle/Entity/Register.php (hunk 6 above)
Comment (mateusz-blachacz) [reply code/src/Eduweb/TrainingBundle/Entity/Register.php]: Thx, for info in the course, there was no information about this

**Review by gracjan122 — APPROVED** [review]
